code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -176,7 +176,17 @@
 	var/datum/language/species_language = GLOB.all_languages[language]
 	return species_language.get_random_name(gender)
 
-/datum/species/proc/create_organs(mob/living/carbon/human/H) //Handles creation of mob organs.
+/datum/species/proc/create_organs(mob/living/carbon/human/H, special = FALSE) //Handles creation of mob organs.
```

Comment:
`special` doesn't tell much. Are the organs created special? In what way?
`include_special_organs` or something would most likely fit better

Reply:
named better